smoke/README.md

```diff
@@ -0,0 +1,105 @@
+# QA for OpenShift Shared Resource CSI Driver
+This is a Behavior-driven development (or BDD) test framework leveraging `behave` for `python`. BDD is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.Behave uses tests written in a natural language style, backed up by Python code.
+
+We are using this framework to achieve the QA verification of any Jira Stories development takes on for the OpenShift Shared Resource CSI Driver, as well as any regression test suites the team wants to cultivate for the OpenShift Shared Resource CSI Driver.
+
+Hence, the reader should consider these tests the 'next phase' of testing after the [golang based tests](https://github.com/openshift/csi-driver-shared-resource/tree/master/test/e2e) that run in OpenShift CI prow jobs as part of merging pull requests with code changes.
```

Comment:
```suggestion
Hence, the reader should consider these tests the 'next phase' of testing after the [golang based tests](https://github.com/openshift/csi-driver-shared-resource/tree/master/test/e2e) that run in OpenShift CI prow jobs as part of merging pull requests with code changes.

Both QA and development have agreed to utilize this framework, and the description of test scenarios detailed below, as a replacement for direct communication between QA and development in OpenShift's internal polarion server.  QA will separately deal with taking the test scenarios descriptions here and importing into polarion for maintaining compatibility with the existing processes of the broader QA organization. 
```

Reply:
Personally, I would omit the suggested second paragraph. This references processes that are more internal to Red Hat and aren't germane to a customer or end user. I propose the following:

```suggestion
Hence, the reader should consider these tests the 'next phase' of testing after the [golang based tests](https://github.com/openshift/csi-driver-shared-resource/tree/master/test/e2e) that run in OpenShift CI prow jobs as part of merging pull requests with code changes. These provide another means of testing that are focused on the end user experience, in language and terms that end users can understand.
```

Reply:
If we are not going to document my suggestion on our team's evolving process here in this README, then I think I'll insist that it is documented *somewhere*

Perhaps https://source.redhat.com/groups/public/atomicopenshift/atomicopenshift_wiki/team_jenkinsbuild_api @adambkaplan ?  Or do you have another suggestion?

@rgarg1 @shwetha-h-p FYI

Reply:
Is there a team confluence page that exists? That might be an apt place to document internal information.

Reply:
no we don't do confluence for this team that I'm aware of

the source wiki I noted above is what we have

we'll discuss in our team meeting most likely and make a decision there one way or the other